src/main/java/net/sf/jabref/gui/EntryTypeDialog.java

```diff
@@ -148,6 +174,95 @@ private JPanel createEntryGroupPanel(String groupTitle, Collection<EntryType> en
         return panel;
     }
 
+    private JPanel createIdFetcher(String groupTitle) {
```

Comment:
Rename to "createIdFetcherPanel" and remove parameter. Just pass it directly to the method below and use Localization.lang to have the title translated into other languages.